src/components/frame/v5/pages/ContributorsPage/ContributorsPage.tsx

```diff
@@ -2,11 +2,44 @@ import React, { FC } from 'react';
 
 import Navigation from '~v5/common/Navigation';
 import TwoColumns from '~v5/frame/TwoColumns';
+import MembersList from '~v5/common/MembersList';
+import TeamReputationSummary from '~v5/common/TeamReputationSummary';
+import { teamsWithSummedUpData } from '~v5/common/TeamReputationSummary/consts';
+import { useSearchContext } from '~context/SearchContext';
+import { useMembersPage } from '../MembersPage/hooks';
+import Header from '~frame/v5/Header';
+import Spinner from '~v5/shared/Spinner';
 
-const ContributorsPage: FC = () => (
-  <TwoColumns aside={<Navigation pageName="members" />}>
-    Contributors page
-  </TwoColumns>
-);
+const ContributorsPage: FC = () => {
+  const { searchValue } = useSearchContext();
+  const { contributors, loading } = useMembersPage(searchValue);
+
```

Comment:
maybe will be better to create one component to display contributors and followers? you can do it by checking the routing url "/colony/colonya/contributors"